### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by nikhilzambare24**
hi please can you review my changes..Please note the changes are in fetch URL's

### CODE REVIEW

File: DisplaySelected.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import { Button, Table } from "react-bootstrap";
+import { useNavigate, useParams } from "react-router-dom";
+
+export default function DisplaySelected() {
+  const [record, setRecord] = useState({});
+  const { url, code } = useParams();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const apiUrl = `http://localhost:8080/api/admin/${url}/${code}`;
```
Comment (HitendraMhatre) [inline DisplaySelected.js:11]: URL are checked and approved based on the latest backend.

Comment (nikhilzambare24) [reply DisplaySelected.js:11]: Ok.

**Review by HitendraMhatre — APPROVED** [review]
URL are as required. Proceed for merge.

File: DeleteSelected.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  }, [url, code]);
+
+  const handleDelete = async () => {
+    const confirmed = window.confirm(
+      "Are you sure you want to delete the selected record?"
+    );
+    if (!confirmed) {
+      return;
+    }
+    const res = await fetch(`http://localhost:8080/api/admin/${url}/${code}`, {
```
Comment (OmkarPatil1950) [inline DeleteSelected.js:27]: The url is matching with backend and working fine as per backend.

**Review by Shruti181807 — APPROVED** [review]
All url are checked with backend . You can proceed with merge.

**Review by OmkarPatil1950 — APPROVED** [review]
all url are correct so you can proceed with merge.

**Comment by nikhilzambare24** [comment]
changes merged  into master

**Pull request merged**